ts-testing-training/src/0_numberStack/index.ts

```diff
@@ -30,13 +30,13 @@ export class NumberStack {
    * Sets up a new NumberStack object.  Is called whenever `new NumberStack()`
    * is called.
    */
-  constructor() {
-    this.list = [];
+  constructor(input: number[] = []) {
+    this.list = input;
   }
 
   /** Puts a number on the top of the stack */
   public push(input: number): void {
-    return;
+    this.list.push(input);
```

Comment:
Have you considered using the spread operator instead?
```suggestion
    this.list = [...this.list, input];
```

Reply:
reply